app/src/main/java/org/simple/clinic/remoteconfig/RemoteConfigSync.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.remoteconfig
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.BuildConfig
+import org.simple.clinic.sync.BatchSize
+import org.simple.clinic.sync.ModelSync
+import org.simple.clinic.sync.SyncConfig
+import org.simple.clinic.sync.SyncGroup
+import org.simple.clinic.sync.SyncInterval
+import timber.log.Timber
+import javax.inject.Inject
+
+class RemoteConfigSync @Inject constructor(private val remoteConfig: FirebaseRemoteConfig) : ModelSync {
+
+  override fun sync(): Completable = pull()
+
+  override fun push(): Completable = Completable.complete()
+
+  override fun pull(): Completable {
+    // Firebase throttles network calls to 5 per hour.
+    // Source: https://firebase.google.com/docs/remote-config/android
+    val cacheExpiration = if (BuildConfig.DEBUG) 0 else SyncInterval.FREQUENT.frequency.seconds
```

Comment:
Can we inject this as a `Duration` instead of hardcoding? Then override it in `DebugClinicApp`?

Reply:
Oh yea, why did I not think of that. I'll do this.

Reply:
Done in the last commit: https://github.com/simpledotorg/simple-android/pull/568/commits/5f378174d491f71846d87fc44e4319e77c16e43f